### PR TITLE
Add Windows ARM64 to official builds

### PR DESCRIPTION
**Pull request by danmosemsft**
@ulisesh tells me the queues are Windows.10.Arm64.Open and Windows.10.Arm64

I kicked off an official build against the private fork, with default settings. buildId=112683

@safern does this look right?

### CODE REVIEW

File: eng/pipelines/windows.yml [hunk 1]
```diff
@@ -63,6 +63,13 @@ jobs:
               _helixQueues: $(netcoreappWindowsQueues)
               _publishTests: true
 
+            arm64_Release:
+              _BuildConfig: Release
+              _architecture: arm64
+              _framework: netcoreapp
+              _helixQueues: $(windowsArmQueue)
+              _publishTests: true
```
Comment (danmosemsft) [inline eng/pipelines/windows.yml]: Not sure whether to include this line or not.

**Comment by danmosemsft** [comment]
Hmm, not really sure how this works, but in the build I queued the new Windows ARM64 passed (green) even though not all the test jobs for that column finished
https://mc.dot.net/#/user/dotnet-mc-bot-2/official~2Fdotnet~2Fcorefx~2Frefs~2Fheads~2Farm64/test~2Ffunctional~2Fcli~2F/20190302.5

**Comment by danmosemsft** [comment]
It seems it is actually running on Windows ARM64 which is positive:
```
### CONFIGURATION: WindowsProductType=4 WindowsInstallationType=Client OS=Microsoft Windows 10.0.17133 OSVer=Microsoft Windows NT 10.0.17133.0 OSArch=Arm64 Arch=Arm64 Framework=.NET Core 9.9.9 LibcRelease=glibc_not_found LibcVersion=glibc_not_found
```
from on the secret S.R.I.R output https://mc.dot.net/#/user/dotnet-mc-bot-2/official~2Fdotnet~2Fcorefx~2Frefs~2Fheads~2Farm64/test~2Ffunctional~2Fcli~2F/20190302.5/workItem/System.Runtime.InteropServices.RuntimeInformation.Tests/wilogs

**Comment by danmosemsft** [comment]
Almost all passing, which is great. Some Drawing ones failed with low order arithmetic diffs eg
```
Expected: {X=2.999624,Y=2.0137072,Width=0,Height=0.013704777}
Actual:   {X=2.9996243,Y=2.0137074,Width=0,Height=0.013704777}
```

Bunch of perf counters issues 
https://mc.dot.net/#/user/dotnet-mc-bot-2/official~2Fdotnet~2Fcorefx~2Frefs~2Fheads~2Farm64/test~2Ffunctional~2Fcli~2F/20190302.5/workItem/System.Diagnostics.PerformanceCounter.Tests

File: eng/pipelines/windows.yml (hunk 1 above)
Comment (safern) [reply eng/pipelines/windows.yml]: This is to publish tests assets to azure blob storage for coreclr to run corefx tests in their CI. I don't think it is necessary now, but might be in the future.

**Comment by safern** [comment]
There is something weird going on with the branch/commit resolution:

```
/eng/pipelines/windows.yml: Unable to find file /eng/pipelines/corefx-base.yml in repository self using ref refs/pull/35721/merge and commit 5fcc8dfcc67c437efd3d27cdad91711da7a28b8f: Not Found
```

That is the error the CI is giving. I wonder why as the yml file looks OK. Did you indent with spaces?

File: eng/pipelines/windows.yml (hunk 1 above)
Comment (safern) [reply eng/pipelines/windows.yml]: cc: @mikem8361 do we want to publish assets for arm64 for coreclr testing?

**Comment by safern** [comment]
The commit it is looking for is not in your tree: 5fcc8dfcc67c437efd3d27cdad91711da7a28b8f

**Comment by danmosemsft** [comment]
Sorry I don't follow?

The official build I did was on a branch/commit separate to this one obviously.

**Comment by safern** [comment]
I’m speaking about corefx-ci build on this PR. Since you’re changing the yml files we need a passing CI.

**Comment by danmosemsft** [comment]
It seems that https://github.com/dotnet/corefx/commit/5fcc8dfcc67c437efd3d27cdad91711da7a28b8f does exist in that the URL resolves -- and it is the candidate hash this PR is testing. If I fetch though, it does'nt show up locally. I am not sure how github implements PR's behind the scenes but it doesn't surprise me that the subject of a PR is not pushed into the main repo before it is merged. It must be in github's "private" copy of the repo. URLs such as the above would work for both, but it wouldn't pull down.

that doesn't explain why the CI can't see it here since normally it works.

**Comment by mikem8361** [comment]
Yes, I think we do want to publish arm64 test assets.

**Comment by danmosemsft** [comment]
@safern it is green now. Also the internal build is. Is any other validation required?

**Comment by safern** [comment]
Should be good. Thanks for adding this!

**Comment by danmosemsft** [comment]
Thanks for your help @safern

**Pull request merged**